pyccel/ast/class_defs.py

```diff
@@ -5,12 +5,14 @@
 """
 This module contains all types which define a python class which is automatically recognised by pyccel
 """
+
+from pyccel.ast.builtin_objects.list_functions import (ListPop)
```

Comment:
```suggestion
from pyccel.ast.builtin_objects.list_functions import ListPop
```
These parentheses do nothing

Reply:
Fixed 8d649bfdb127979ea3223f0bfc80ba57ca5efacc